core/src/test/java/com/pholser/junit/quickcheck/runner/PropertyFalsifiedUtilityClassTest.java

```diff
@@ -26,11 +26,119 @@ a copy of this software and associated documentation files (the
 package com.pholser.junit.quickcheck.runner;
 
 import com.pholser.junit.quickcheck.UtilityClassesUninstantiabilityHarness;
```

Comment:
I don't use `*` imports in the project except in `static` imports. Please expand this out.